### PR TITLE
Adding Get-CalendarDiagnosticObjectsSummary.ps1

### PR DESCRIPTION
**Pull request by EnzoLopez2023**
Issue:
This script runs the Get-CalendarDiagnosticObjects script and returns a summarized timeline of actions into clear english

Reason:
There's no script available on the repo to achieve this

Validation:
Have validated this script on a test environment and it's also being used as an internal tool

### CODE REVIEW

**Comment by dpaulson45** [comment]
@EnzoLopez2023 from now on, let's not close and open a new PR when we state we have changes. We have the ability to rebase the branch to clean up the commits if that is what you are worried about. It makes it harder for review otherwise.

**Comment by dpaulson45** [comment]
Code formatting did not pass with the following: 

image

Please address.

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 [hunk 1]
```diff
@@ -0,0 +1,922 @@
...
+        $script:GetCDO = Get-CalendarDiagnosticObjects -Identity $Identity -MeetingID $MeetingID -CustomPropertyNames $CustomPropertyNameList -WarningAction Ignore -MaxResults 2000;
+    }
+
+    if ($Identity -and !$Subject -and !$MeetingID) {
+        Write-Warning "Can't run command with just Identity, Subject or MeetingID is also needed";
+        exit;
+    }
+}
+
+GetCalendarDiagnosticObjects;
```
Comment (dpaulson45) [inline Calendar/Get-CalendarDiagnosticObjectsSummary.ps1:51]: This should move down towards the bottom of script, so it isn't in the middle of a bunch of function declares and hidden.

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 [hunk 2]
```diff
@@ -0,0 +1,922 @@
...
+    }
+} elseif ($UniqueMeetingID.count -eq 1) {
+    $GCDO = $GetCDO;
+    $GetCDO = @();
+    BuildCSV;
+    BuildTimeline;
+} else {
+    Write-Warning "A valid meeting ID was not found, manually confirm the meetingID";
+}
+$TimeSum = @();
```
Comment (dpaulson45) [inline Calendar/Get-CalendarDiagnosticObjectsSummary.ps1:922]: This needs to be defined prior to being used in `Convert-Data` which is called by `BuildTimeline->MeetingSummary->Convert-Data`. So this should be declared prior to `BuildTimeline` is called.

**Review by dpaulson45 — CHANGES_REQUESTED** [review]

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 (hunk 1 above)
Comment (EnzoLopez2023) [reply Calendar/Get-CalendarDiagnosticObjectsSummary.ps1]: Moved down after last function.

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 (hunk 2 above)
Comment (EnzoLopez2023) [reply Calendar/Get-CalendarDiagnosticObjectsSummary.ps1]: Removed, was not needed

**Comment by dpaulson45** [comment]
image


prior to commit, run `.build\CodeFormatter.ps1` to make sure your commits aren't going to fail in the pipeline.

**Review by dpaulson45 — APPROVED** [review]

**Pull request merged**